logging/config/cluster-logging-configuring-cr.adoc

```diff
@@ -8,9 +8,35 @@ toc::[]
 
 To configure {logging-title} you customize the `ClusterLogging` custom resource (CR).
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
-
 include::modules/cluster-logging-about-crd.adoc[leveloffset=+1]
+
+////
+// collecting this information here for a future PR
+
+If you want to specify collector resources or scheduling, you must create a `ClusterLogging` CR:
+
+.ClusterLogging resource example
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: audit-collector <1>
+  namespace: openshift-kube-apiserver <2>
+spec:
+  collection:
+    type: "vector" <3>
+    resources:
+      limits:
+        memory: 2G
+# ...
+----
+<1> The name of the `ClusterLogging` CR must be the same as the `ClusterLogForwarder` CR.
+<2> The namespace of the `ClusterLogging` CR must be the same as the `ClusterLogForwarder` CR.
+<3> The collector type that you want to use. This example uses the Vector collector.
```

Comment:
`vector` is the only supported collector when deploying multiple-CLF, I think we have to write this in our document.